ui-admin/src/forms/FormContentEditor.tsx

```diff
@@ -8,6 +8,7 @@ import { OnChangeFormContent } from './formEditorTypes'
 import { FormContentJsonEditor } from './FormContentJsonEditor'
 import { FormPreview } from './FormPreview'
 import { validateFormContent } from './formContentValidation'
+import ErrorBoundary from 'util/ErrorBoundary'
```

Comment:
Nothing in this file actually changed other than wrapping the components in ErrorBoundary.

View without whitespace changes https://github.com/broadinstitute/juniper/pull/518/files?w=1